content/en/getting_started/site/_index.md

```diff
@@ -13,18 +25,58 @@ Datadog offers different sites throughout the world. Each site is completely ind
 | EU1     | `https://app.datadoghq.eu`  | `datadoghq.eu`      | EU       |
 | US1-FED | `https://app.ddog-gov.com`  | `ddog-gov.com`      | US       |
 
-## How do I tell which Datadog site I am on?
+All Datadog traffic is transmitted over SSL (default 443) to the following domains:
+
+{{< tabs >}}
+{{< tab "Logs" >}}
+
+| Site | Site URL                                      |
+|------|-----------------------------------------------|
+| US1  | https://logs.browser-intake-datadoghq.com     |
+| US3  | https://logs.browser-intake-us3-datadoghq.com |
+| US5  | https://logs.browser-intake-us5-datadoghq.com |
+| EU1  | https://mobile-http-intake.logs.datadoghq.eu  |
+
+{{< /tab >}}
+{{< tab "Traces" >}}
+
+| Site | Site URL                                           |
+|------|----------------------------------------------------|
+| US1  | https://trace.browser-intake-datadoghq.com         |
+| US3  | https://trace.browser-intake-us3-datadoghq.com     |
+| US5  | https://trace.browser-intake-us5-datadoghq.com     |
+| EU1  | https://public-trace-http-intake.logs.datadoghq.eu |
+
+{{< /tab >}}
+{{< tab "RUM" >}}
+
+| Site | Site URL                                     |
+|------|----------------------------------------------|
+| US1  | https://rum.browser-intake-datadoghq.com     |
+| US3  | https://rum.browser-intake-us3-datadoghq.com |
+| US5  | https://rum.browser-intake-us5-datadoghq.com |
+| EU1  | https://rum-http-intake.logs.datadoghq.eu    |
+
+{{< /tab >}}
+{{< /tabs >}}
+
+## Navigate the Docs site 
+
+Different Datadog sites may support different functionalities depending on the instance's security requirements. You can use the site selector dropdown menu on the right side of the Docs site to select the Datadog site you want to see information about.
+
+{{< img src="getting_started/site/site-selector.png" alt="The site selector dropdown menu on the right hand side of the Documentation site" style="width:100%" >}}
 
-You can identify which site you are on by matching your Datadog website URL to the site URL in the table above.
+For example, to access information about the Datadog for Government site, select **US1-FED**.
```

Comment:
```suggestion
For example, to see the documentation for the Datadog for Government site, select **US1-FED**.
```

being super specific here that this shows different documentation, not just "information", generally